### DISCUSSION TITLE
Change the Name of This Project?

### DISCUSSION THREAD

**Discussion by scorpion-orion** [JanusGraph/janusgraph#2729] [General]
It has been 9 months since I've been using Janus in our projects. We all know J stands for Java right and people at work keep calling this Java-anus or J-anus everyday.

Is there any plan to rename this? Not asking for the mountain to bend over for me but just asking if there's any discussion going on with the name.

**Comment by li-boxuan** [comment]
I personally never heard of anyone relating "Ja" to "Java" and calling "Janus" as "Java-anus", but what you said sounds very fun 😉 
I think the chance of changing the name is very very small because that will damage the branding.

**Reply by krlawrence** [reply]
The J in Janus has nothing to do with Java. Back when the project was being formed it was quite hard to find a good name that also was not already being used. The name Janus was picked as it has two appropriate meanings. Janus is a moon which aligned nicely with Titan also a moon. Janus was also in Greek Mythology the name of a god of "new beginnings". This was a new project building on prior work so the name aligned well people felt.

**Reply by porunov** [reply]
Thank you @krlawrence for this information! I believe Janus name well aligns with the project.

@brandnewx Your interpretation is quite funny :smile:
